integration-tests/src/test/groovy/com/okta/sdk/tests/it/GroupRolesIT.groovy

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright 2020-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.sdk.tests.it
+
+import com.okta.sdk.client.Client
+import com.okta.sdk.resource.group.GroupBuilder
+import com.okta.sdk.resource.group.Group
+import com.okta.sdk.resource.role.AssignRoleRequest
+import com.okta.sdk.resource.role.RoleType
+import com.okta.sdk.resource.user.Role
+import com.okta.sdk.resource.user.RoleList
+import com.okta.sdk.tests.Scenario
+import com.okta.sdk.tests.it.util.ITSupport
+import org.testng.annotations.Test
+
+import static com.okta.sdk.tests.it.util.Util.assertGroupPresent
+import static com.okta.sdk.tests.it.util.Util.assertGroupAbsent
+import static com.okta.sdk.tests.it.util.Util.assertRolePresent
+import static com.okta.sdk.tests.it.util.Util.assertRoleAbsent
+import static com.okta.sdk.tests.it.util.Util.validateGroup
+import static org.hamcrest.MatcherAssert.assertThat
+import static org.hamcrest.Matchers.*
+
+/**
+ * Tests for /api/v1/groups/roles
+ * @since 2.0.0
+ */
+class GroupRolesIT extends ITSupport implements CrudTestSupport {
+
+    @Override
+    def create(Client client) {
+        return GroupBuilder.instance()
```

Comment:
Is this creating a Group? or a GroupRole?

Reply:
Got rid of this as per your other comment.